hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java

```diff
@@ -31,6 +31,9 @@
      * @param password Password for the user.
      * @param uuid Unique string identifying the connected client uniquely. This string is generated by the owner member server
      *             on initial connection. When the client connects to a non-owner member it sets this field on the request.
+     *             A UUID represents a 128-bit value. This string is a randomly generated uid based on
```

Comment:
This also looks like added back when merging. It is better to remove detailed explanation related to UUID